### ISSUE TITLE
Can't be installed with Yarn

### ISSUE THREAD

**Issue by nigelgutzmann** [pixijs/webdoc-template#12]
When I try to install this package with yarn, I get the following error:

```
nigel$ mkdir new_project
nigel$ cd new_project/
nigel$ yarn add @pixi/webdoc-template
yarn add v1.22.10
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
warning @pixi/webdoc-template@1.3.3: The engine "pnpm" appears to be invalid.
error @pixi/webdoc-template@1.3.3: The engine "yarn" is incompatible with this module. Expected version "please-use-npm". Got "1.22.10"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

The package.json specifies `please-use-npm` in the engines section, but I believe this is only for developers of this package (to avoid developers of this package using both yarn and npm and generating conflicting `package-lock.json` vs `yarn.lock`). It should not be enforced for consumers installing this package in their projects, and should be possible to install `webdoc-template` via yarn

**Comment by nigelgutzmann** [comment]
@stepancar pointed out a workaround to me: 

```
yarn add @pixi/webdoc-template --ignore-engines
```

**Comment by stepancar** [comment]
@nigelgutzmann , temporary solution. This package should work with yarn too. https://github.com/pixijs/sound/issues/163

**Comment by bigtimebuddy** [comment]
Fixed in 1.3.4. Thanks @stepancar for the fix!
